src/java.base/share/classes/sun/net/www/protocol/http/HttpURLConnection.java

```diff
@@ -289,7 +289,7 @@ private static String getNetProperty(String name) {
 
     static final String httpVersion = "HTTP/1.1";
     static final String acceptString =
-        "text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2";
+        "text/html, image/gif, image/jpeg, */*; q=0.2";
 
```

Comment:
It looks syntactically correct now, but I wonder about the actual semantics, and whether it makes any sense to be asking for 80% reduction in quality of other media types.

Reply:
Should we just have `static final String acceptString = "*/*";`